POLICIES.rdoc

```diff
@@ -45,14 +45,31 @@ For example, RubyGems 2.5.0 was recently released.  RubyGems does not
 guarantee any non-security bugfix releases will be made upon the 2.4.x series,
 or any earlier series, of releases.
 
-Additionally, when a Ruby version reaches end-of-life the following release of
-RubyGems will no longer be required to maintain backwards compatibility with
-that Ruby version.  This may result in a major version change.
+Additionally, when a Ruby version reaches end-of-life the following minor
+release of RubyGems will drop backwards compatibility with that Ruby version.
 
-For example, since Ruby 1.9.3 has reached end-of-life, RubyGems may use
-features of ruby that only exist in Ruby 2.0 and later.  As of this writing
-RubyGems is 2.5.x, so the RubyGems version would become 3.x if it used
-features only available in Ruby 2.0 and later.
+For example, since Ruby 2.2 has reached end-of-life, RubyGems may use require
+Ruby 2.3 or later.  As of this writing RubyGems is at versino 3.0, so the
+RubyGems version would become 3.1 if it used features only available in Ruby
+2.3 and later.
+
+=== Security Release Process
+
+- Obtain CVE numbers as needed from HackerOne or RedHat.
```

Comment:
Why `RedHat`? I wonder why they assign CVE number of rubygems.

Reply:
Not sure I understand the question, but if you're wondering why a CVE number on RubyGems and not just Ruby itself (as RubyGems ships with Ruby, of course) it's because Red Hat RPMs packages it's own Ruby very differently than MRI and RubyGems is also a separate package. Don't ask 🙄

Disclaimer: I work for Red Hat

@indirect  s/RedHat/Red Hat/   👈  I have to include this or I'll be fired :trollface:

Reply:
I'll add that if any help or guidance is needed in terms of CVE process for rubygems or any gem for that matter, we can certainly find someone to answer any questions you may have.  I also work for Red Hat but on the application side.

Reply:
@hsbt So far, the Bundler and RubyGems projects have always gotten CVE numbers from either HackerOne or Red Hat's OSS security team. Those companies already have large blocks of CVE numbers assigned from MITRE, and they are willing to grant CVE numbers from their already-assigned blocks to Ruby open source projects. The point of the CVE system ultimately to make sure that every security issue has just one number, so it doesn't really matter whether we get our CVE numbers directly from MITRE or out of a block assigned by MITRE to another company.

Reply:
(Also, it is a lot of work and takes a long time to get a block of CVE numbers assigned directly by MITRE, so it does not seem worth doing that work if we only need ~1-5 CVE numbers per year.)